services/coingecko.ts

```diff
@@ -90,12 +90,12 @@ async function getCoinPrice(search: string, fiat = settings?.currency.defaultFia
   if (!publicRuntimeConfig.enableCoinGecko)
     return 0;
 
-  const storage = new WinStorage(`coingecko:${search.toLowerCase()}`);
+  const storage = new WinStorage(`coingecko:${search?.toLowerCase()}`);
```

Comment:
again: this parameter is set as mandatory, if the function calling this does not have a "search" value, then there's a bug on that function. this function shouldn't have a `? ` on search